aws/ssm/nanny.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+
+curl -O https://raw.githubusercontent.com/harmony-one/harmony/master/scripts/wallet.sh
+
+mkdir /home/ec2-user/$(date +%Y%m%d_%H%M%S)
```

Comment:
mkdir -p

Reply:
This is a script for AWS SSM integration test. Since the python application can successfully invoke a bash script using AWS SSM, this test script becomes obsolete.

Reply:
In the future, I will use `-p` argument to create a chain of folders.